packages/sdk/src/browser-wallets/sats-connect/index.ts

```diff
@@ -0,0 +1,266 @@
+import * as ecc from "@bitcoinerlab/secp256k1";
```

Comment:
Can we move this to `browser-wallets/internal/sats-connect`? The satsConnect prefix being in the browser-wallets folder makes it seem like sats-connect is a browser wallet.

Reply:
Yup, moved.